assets/components/paymentButtons/stripePopUpButton/stripePopUpButton.jsx

```diff
@@ -51,6 +52,15 @@ const StripePopUpButton = (props: PropTypes) => (
   />
 );
 
+function formIsValid(formClassName: string) {
+  const form = document.getElementsByClassName(formClassName);
```

Comment:
when I know I just want one element, I'd tend to do `querySelector` and just `if (form) {` since it'll get the first one, or return null